spec/raven/instance_spec.rb

```diff
@@ -219,6 +219,43 @@ def ivars(object)
     end
   end
 
+  describe '.user_context' do
+    context 'with `options` that is a hash' do
```

Comment:
Arguments can be made whether the context is right given that the check is `respond_to?(:has_key?)`, but lets be real here - this is a Hash structure.